app/assets/stylesheets/runtime-banner.scss

```diff
@@ -0,0 +1,15 @@
+.runtime-banner {
+  position: fixed;
+  left: 0;
+  right: 0;
+  bottom: calc(65px + env(safe-area-inset-bottom));
+  z-index: var(--z-sticky);
+
+  &__inner {
+    background-color: #0b0c0c;
+    color: #fff;
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+  }
+}
```

Comment:
These are all placeholder values that will be fleshed out in a separate PR when implementing [RFC 72](https://github.com/forem/rfcs/pull/72). None of this is visible because of the `runtime_banner` Feature Flag